docs/source/course/index.md

```diff
@@ -4,6 +4,11 @@ If you like to learn from video, you can follow our [hands-on course "Introducti
 
 The course is structured into sections and these are each broken into short videos that cover specific Kedro topics. You'll walk through the [spaceflights tutorial](../tutorial/spaceflights_tutorial.md) and get hands-on with the example. Along the way, you'll learn key Kedro concepts like datasets and the Kedro Data Catalog, nodes and pipelines, and configuration management.
 
+```{eval-rst}
+..  youtube:: DD7JuYKp6BA
+    :width: 90%
```

Comment:
I think you could make these widths `100%`, or something closer to it. When I checked on mobile it looked a little bit off that it wasn't filling the whole space like the content above and below.